server/settings.py

```diff
@@ -221,3 +223,11 @@
 KEYWORDS_ADD_MISSING_ON_PUBLISH = True
 MACROS_MODULE = env('MACROS_MODULE', 'belga.macros')
 
+SECRET_KEY = 'SomethingNotEntirelySecret'
+OIDC_CLIENT_SECRETS = env('OIDC_CLIENT_SECRETS', 'client_secrets.json')
```

Comment:
would be good to use env variables for all secrets, also for client secret parts:
```py
SECRET_KEY = env('SECRET_KEY')
OIDC_CLIENT_SECRETS = {
    'web': {
        'client_id: env('OIDC_CLIENT_ID'),
        'client_secret': env('OIDC_CLIENT_SECRET'),
    },
}
```
or maybe just define those `OIDC_CLIENT_SECRET` etc and construct the config using these in init_app:
```py
def init_app(app):
    app.config.setdefault('OIDC_CLIENT_SECRETS', {
        'web': {
             'client_secret: app.config.get('OIDC_CLIENT_SECRET'),
             ...,
        },
    })
```

Reply:
Unfortunately, there is no release from commit https://github.com/puiterwijk/flask-oidc/commit/de9584677fbed8f8688eab60145744cb353915ea which support loading `CLIENT_SECRET` from dict, we are considering [Flask-OIDC-ex](https://github.com/larsw/flask-oidc-ex), if it's working fine, we will update accordingly.